pandas/core/indexes/base.py

```diff
@@ -418,7 +418,7 @@ def asi8(self):
         return None
 
     @classmethod
-    def _simple_new(cls, values, name: Label = None):
+    def _simple_new(cls, values: np.ndarray, name: Label = None):
```

Comment:
because this is the base class it should be ArrayLike? (or we can leave this fix for #36092)

Reply:
no, its specifically np.ndarray

Reply:
hmm, I guess you want to make Index np.ndarray specific and have ExtensionIndex handle EA backed indexes but in that case some other type annotations in base would need to be changed e,g, `_data: Union[ExtensionArray, np.ndarray]` so that methods in the base class that currently handle EA can be simplified and overridden.

In #37898, im getting `error: Unsupported left operand type for - ("ExtensionArray")` in the base class since  `def _values(self) -> Union[ExtensionArray, np.ndarray]` so atm the base class is not EA agnostic.

Reply:
updated to get revert this annotation